src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -16,11 +16,11 @@ $(document).ready(function() {
      * Handles Keyup and Keydown on Text question to display response length
      */
     $('textarea[id^="responsetext-"]').keyup(function() {
```

Comment:
Not entirely related to this PR, but perhaps you could use `.on('input', function() {...})` instead of `.keyup` and `.keydown`. That way, it also factors in the cases where the user cuts or pastes into the `textarea` with a mouse. Create a separate issue for it maybe?